operator/v1/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -331,6 +331,16 @@ spec:
                         type: integer
                         format: int32
                         minimum: 0
+                      v4JoinSubnet:
```

Comment:
Did you consider putting them in a sub-struct? e.g. Could there be other configuration needed for this?

i.e.
ovnKubernetesConfig:
    joinSubnets:
        v4: 100.65.0.0/16

Reply:
I don't anticipate any other configuration which can fall under joinSubnets to make this as a sub-struct property.